node/node.go

```diff
@@ -1960,7 +1961,12 @@ func (app *App) verifyDB(ctx context.Context) {
 			}
 
 			// verify atx signature
-			if !app.edVerifier.Verify(signing.ATX, atx.SmesherID, atx.SignedBytes(), atx.Signature) {
+			// TODO: use atx handler to verify signature
```

Comment:
This PR or next? 🙂

Reply:
TBH, I'd remove this function from `develop` as it's unused and keep it on a branch (maybe in a draft PR).

Reply:
You can remove it if you want - it can be re-added if needed